app/models/language_skill.rb

```diff
@@ -1,6 +1,13 @@
 class LanguageSkill < ApplicationRecord
   belongs_to :languageable, polymorphic: true, optional: true
 
+  scope :native, -> { where(level: 'native_speaker') }
+  scope :native_languages, lambda {
+    return native if native.size <= 1
+    native.order("CASE WHEN language = 'DE' THEN 1 WHEN language != 'DE' THEN 2 END")
+  }
+  scope :non_native_languages, -> { where.not(id: native_languages.first&.id) }
```

Comment:
Should this actually only consider the first `native_languages`? Implying from the naming this should return the complementing set to `native_languages`.